### ISSUE TITLE
Unable to build an image using latest code.

### ISSUE THREAD

**Issue by psachin** [rhinstaller/lorax#102]
I don't know if this is know issue or not,

`FileNotFoundError: [Errno 2] No such file or directory: '/root/results/images/install.img'`

More logs
```

2016-03-19 18:50:18,745: populating output tree and building boot images
populating output tree and building boot images
2016-03-19 18:50:18,849: Problem rendering x86.tmpl ({'glob': <function LoraxTemplateRunner.__init__.<locals>.<lambda> at 0x7f45788ffd90>, 'inroot': '/var/tmp/lorax.cgmhvmc1/installroot', 'domacboot': True, 'lower': <function string_lower at 0x7f45788e30d0>, 'udev': <function udev_escape at 0x7f45788e3048>, 'isolabel': 'Fedora-23-x86_64', 'product': {'variant': None, 'isfinal': False, 'name': 'Fedora', 'bugurl': 'your distribution provided bug reporting tool', 'version': '23', 'release': '23'}, 'outroot': '/root/results', 'basearch': 'x86_64', 'libdir': 'lib64', 'kernels': [{'path': 'boot/vmlinuz-4.4.5-300.fc23.x86_64', 'initrd': {'path': 'boot/initramfs-4.4.5-300.fc23.x86_64.img'}, 'version': '4.4.5-300.fc23.x86_64', 'flavor': None, 'arch': 'x86_64'}], 'arch': {'basearch': 'x86_64', 'libdir': 'lib64', 'bcj': 'x86', 'buildarch': 'x86_64'}, 'runtime_img': 'images/install.img', 'exists': <function LoraxTemplateRunner.__init__.<locals>.<lambda> at 0x7f45788ff6a8>, 'workdir': '/var/tmp/lorax.cgmhvmc1', 'runtime_base': 'install.img', 'doupgrade': True}):
Problem rendering x86.tmpl ({'glob': <function LoraxTemplateRunner.__init__.<locals>.<lambda> at 0x7f45788ffd90>, 'inroot': '/var/tmp/lorax.cgmhvmc1/installroot', 'domacboot': True, 'lower': <function string_lower at 0x7f45788e30d0>, 'udev': <function udev_escape at 0x7f45788e3048>, 'isolabel': 'Fedora-23-x86_64', 'product': {'variant': None, 'isfinal': False, 'name': 'Fedora', 'bugurl': 'your distribution provided bug reporting tool', 'version': '23', 'release': '23'}, 'outroot': '/root/results', 'basearch': 'x86_64', 'libdir': 'lib64', 'kernels': [{'path': 'boot/vmlinuz-4.4.5-300.fc23.x86_64', 'initrd': {'path': 'boot/initramfs-4.4.5-300.fc23.x86_64.img'}, 'version': '4.4.5-300.fc23.x86_64', 'flavor': None, 'arch': 'x86_64'}], 'arch': {'basearch': 'x86_64', 'libdir': 'lib64', 'bcj': 'x86', 'buildarch': 'x86_64'}, 'runtime_img': 'images/install.img', 'exists': <function LoraxTemplateRunner.__init__.<locals>.<lambda> at 0x7f45788ff6a8>, 'workdir': '/var/tmp/lorax.cgmhvmc1', 'runtime_base': 'install.img', 'doupgrade': True}):
2016-03-19 18:50:19,052: 

Traceback (most recent call last):
  File "/usr/lib/python3.4/site-packages/pylorax/ltmpl.py", line 55, in parse
    textbuf = template.render(**variables)
  File "/usr/lib/python3.4/site-packages/mako/template.py", line 443, in render
    return runtime._render(self, self.callable_, args, data)
  File "/usr/lib/python3.4/site-packages/mako/runtime.py", line 803, in _render
    **_kwargs_for_callable(callable_, data))
  File "/usr/lib/python3.4/site-packages/mako/runtime.py", line 835, in _render_context
    _exec_template(inherit, lclcontext, args=args, kwargs=kwargs)
  File "/usr/lib/python3.4/site-packages/mako/runtime.py", line 860, in _exec_template
    callable_(context, *args, **kwargs)
  File "/usr/share/lorax/templates.d/99-generic/x86.tmpl", line 19, in render_body
    <%
FileNotFoundError: [Errno 2] No such file or directory: '/root/results/images/install.img'



Traceback (most recent call last):
  File "/usr/lib/python3.4/site-packages/pylorax/ltmpl.py", line 55, in parse
    textbuf = template.render(**variables)
  File "/usr/lib/python3.4/site-packages/mako/template.py", line 443, in render
    return runtime._render(self, self.callable_, args, data)
  File "/usr/lib/python3.4/site-packages/mako/runtime.py", line 803, in _render
    **_kwargs_for_callable(callable_, data))
  File "/usr/lib/python3.4/site-packages/mako/runtime.py", line 835, in _render_context
    _exec_template(inherit, lclcontext, args=args, kwargs=kwargs)
  File "/usr/lib/python3.4/site-packages/mako/runtime.py", line 860, in _exec_template
    callable_(context, *args, **kwargs)
  File "/usr/share/lorax/templates.d/99-generic/x86.tmpl", line 19, in render_body
    <%
FileNotFoundError: [Errno 2] No such file or directory: '/root/results/images/install.img'

Traceback (most recent call last):
  File "/home/sacpatil/github/lorax/src/sbin/lorax", line 353, in <module>
    main(sys.argv)
  File "/home/sacpatil/github/lorax/src/sbin/lorax", line 209, in main
    remove_temp=True, verify=opts.verify)
  File "/usr/lib/python3.4/site-packages/pylorax/__init__.py", line 355, in run
    treebuilder.build()
  File "/usr/lib/python3.4/site-packages/pylorax/treebuilder.py", line 301, in build
    self._runner.run(templatefile, kernels=self.kernels)
  File "/usr/lib/python3.4/site-packages/pylorax/ltmpl.py", line 218, in run
    commands = t.parse(templatefile, variables)
  File "/usr/lib/python3.4/site-packages/pylorax/ltmpl.py", line 55, in parse
    textbuf = template.render(**variables)
  File "/usr/lib/python3.4/site-packages/mako/template.py", line 443, in render
    return runtime._render(self, self.callable_, args, data)
  File "/usr/lib/python3.4/site-packages/mako/runtime.py", line 803, in _render
    **_kwargs_for_callable(callable_, data))
  File "/usr/lib/python3.4/site-packages/mako/runtime.py", line 835, in _render_context
    _exec_template(inherit, lclcontext, args=args, kwargs=kwargs)
  File "/usr/lib/python3.4/site-packages/mako/runtime.py", line 860, in _exec_template
    callable_(context, *args, **kwargs)
  File "x86_tmpl", line 46, in render_body
FileNotFoundError: [Errno 2] No such file or directory: '/root/results/images/install.img'
```

Trying to run

```
lorax -p Fedora -v 23 -r 23 \
      -s http://dl.fedoraproject.org/pub/fedora/linux/releases/23/Everything/x86_64/os/ \
      -s http://dl.fedoraproject.org/pub/fedora/linux/updates/23/x86_64/ \
      ./results/

```

**Comment by bcl** [comment]
Fixed by #104

**Comment by psachin** [comment]
Works now! @bcl :+1:   @dashea :+1:
